<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateLibEditView</class>
 <widget class="QWidget" name="TemplateLibEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="FocAndLev">
      <attribute name="title">
       <string>FocAndLev</string>
      </attribute>
      <widget class="GroupBox" name="ill_groupBox1">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>20</y>
         <width>400</width>
         <height>360</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>45</number>
        </property>
        <item row="0" column="0">
         <widget class="IllustrationView" name="illustration1" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="GroupBox" name="fl_groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>455</width>
         <height>322</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>45</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <property name="spacing">
         <number>15</number>
        </property>
        <item row="2" column="10">
         <widget class="QPushButton" name="fl_up_pushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>up</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QPushButton" name="fl_add_pushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>passThreshold</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="QPushButton" name="fl_delete_pushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
        <item row="3" column="10">
         <widget class="QPushButton" name="fl_down_pushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>down</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spb_FL_PassThreshold">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="4" colspan="2">
         <widget class="QTableWidget" name="fl_tableWidget">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>200</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Align">
      <attribute name="title">
       <string>Align</string>
      </attribute>
      <widget class="GroupBox" name="ill_groupBox2">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>20</y>
         <width>400</width>
         <height>360</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>500</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="topMargin">
         <number>45</number>
        </property>
        <item row="0" column="0">
         <widget class="IllustrationView" name="illustration2" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="GroupBox" name="gpb_align_flpoint">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>526</width>
         <height>280</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_82">
        <property name="topMargin">
         <number>45</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item row="0" column="0" rowspan="4" colspan="2">
         <widget class="QTableWidget" name="align_tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>100</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>80</number>
          </attribute>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>MarkId</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Threshold</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="align_up_pushbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>up</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="align_down_pushbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>down</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="align_add_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="align_delete_pushbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>passThreshold</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spb_Align_PassThreshold">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Size">
      <attribute name="title">
       <string>Size</string>
      </attribute>
      <widget class="GroupBox" name="size_groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>341</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QComboBox" name="cmb_Shape">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Rect</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="lbl_Shape">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>shape</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_ThicknessUnit">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>190</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_Width">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>width</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_Height">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>height</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_HeightUnit">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>140</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="sBox_Height">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>140</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_Thickness">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>thickness</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_WidthUnit">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>90</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="sBox_Thickness">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="sBox_Width">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
       </widget>
      </widget>
      <widget class="GroupBox" name="ill_groupBox3">
       <property name="geometry">
        <rect>
         <x>365</x>
         <y>20</y>
         <width>400</width>
         <height>360</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="topMargin">
         <number>45</number>
        </property>
        <item row="0" column="0">
         <widget class="IllustrationView" name="illustration3" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>CustomerWidget/GroupBox/groupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IllustrationView</class>
   <extends>QWidget</extends>
   <header>CustomerWidget/IllustrationView/illustrationview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
